core/src/main/java/com/sequenceiq/distrox/v1/distrox/converter/DistroXV1RequestToStackV4RequestConverter.java

```diff
@@ -116,13 +117,13 @@ public StackV4Request convert(DistroXV1Request source) {
         request.setGatewayPort(source.getGatewayPort());
         request.setExternalDatabase(getIfNotNull(source.getExternalDatabase(), databaseRequestConverter::convert));
         request.setEnableLoadBalancer(source.isEnableLoadBalancer());
-        calculateVariant(source, request);
+        calculateVariant(environment, source, request);
         checkMultipleGatewayNodes(source);
         return request;
     }
 
-    private void calculateVariant(DistroXV1Request source, StackV4Request request) {
-        if (CloudPlatform.AWS.equals(source.getCloudPlatform()) &&
+    private void calculateVariant(DetailedEnvironmentResponse environment, DistroXV1Request source, StackV4Request request) {
+        if (StringUtils.equals(environment.getCloudPlatform(), "AWS") &&
```

Comment:
why we dont use a simple equals?

Reply:
i like it because it handles null, does it matter if I use this instead of simple equals?